opsdroid/tests/test_logging.py

```diff
@@ -0,0 +1,175 @@
+import pytest
+import logging
+import os
+
+import opsdroid.logging as opsdroid
+from opsdroid.cli.start import configure_lang
+
+from opsdroid.testing.fixtures import _tmp_dir
+
+configure_lang({})
+
+"""Test the logging module."""
+
+
+def test_set_logging_level():
+    assert logging.DEBUG == opsdroid.get_logging_level("debug")
+    assert logging.INFO == opsdroid.get_logging_level("info")
+    assert logging.WARNING == opsdroid.get_logging_level("warning")
+    assert logging.ERROR == opsdroid.get_logging_level("error")
+    assert logging.CRITICAL == opsdroid.get_logging_level("critical")
+    assert logging.INFO == opsdroid.get_logging_level("")
+
+
+def test_configure_no_logging():
+    config = {"logging": {"path": False, "console": False}}
+    opsdroid.configure_logging(config)
+    rootlogger = logging.getLogger()
+    assert len(rootlogger.handlers) == 1
+    assert isinstance(rootlogger.handlers[0], logging.StreamHandler)
+    assert rootlogger.handlers[0].level == logging.CRITICAL
+
+
+def test_configure_file_logging():
+    config = {
+        "logging": {"path": os.path.join(_tmp_dir, "output.log"), "console": False}
+    }
+    opsdroid.configure_logging(config)
+    rootlogger = logging.getLogger()
+    assert len(rootlogger.handlers), 2
+    assert isinstance(rootlogger.handlers[0], logging.StreamHandler)
+    assert rootlogger.handlers[0].level, logging.CRITICAL
+    assert isinstance(rootlogger.handlers[1], logging.handlers.RotatingFileHandler)
+    assert rootlogger.handlers[1].level == logging.INFO
```

Comment:
Instead of using your `_tmp_dir` fixture it would be better to use `tempfile.TemporaryDirectory` in a context manager.

```suggestion
def test_configure_file_logging():
    with tempfile.TemporaryDirectory() as tmpdir:
        config = {
            "logging": {"path": os.path.join(tmpdir, "output.log"), "console": False}
        }
        opsdroid.configure_logging(config)
        rootlogger = logging.getLogger()
        assert len(rootlogger.handlers), 2
        assert isinstance(rootlogger.handlers[0], logging.StreamHandler)
        assert rootlogger.handlers[0].level, logging.CRITICAL
        assert isinstance(rootlogger.handlers[1], logging.handlers.RotatingFileHandler)
        assert rootlogger.handlers[1].level == logging.INFO
```

Reply:
thanks for the tip!
pushed my changes accordingly.